base/doc/ltnews42.tex

```diff
@@ -353,7 +353,17 @@ \subsection{Support separate font families for script fonts}
 conditional code and the condition has changed and therefore requires
 re-initialization.
 
-
+\subsection{Recovering the argument specifier for document commands}
+
+In \LaTeX{} News 38~\cite{42:ltnews38} we detailed that we had \emph{removed}
+\cs{GetDocumentCommandArgSpec} as we felt that it was only required for
+debugging. However, there are some specialist use cases where access to the
+argument specification is useful: see
+\url{https://github.com/latex3/latex3/pull/1799} for example. We have therefore
+looked again at this area and added a \emph{code} interface
+\cs{\detokenize{cmd_arg_spec:N}} for accessing the argument specification. The
+use of a code- rather than design-level name here reflects the fact that this
+is an very specialized use case mainly of interest to package authors.
```

Comment:
and has the potential to slow dow processing a lot if used more generally.

(or something along the lines). This is not just because usa special in my opinion)